src/provision_server.py

```diff
@@ -101,38 +98,43 @@ def print_conf(D, path):
 	# slurm.conf
 	C = parse_slurm_conf("{CPR}/conf/slurm.conf".format(CPR = shlex.quote(CLUST_PROV_ROOT)))
 	C[["ControlMachine", "ControlAddr", "AccountingStorageHost"]] = ctrl_hostname
-	C["SuspendExcNodes"] = ctrl_hostname + "-nfs"
+	del C["SuspendExcNodes"]
 
 	# node definitions
 	C["NodeName8"] = "{HN}-worker[1-100] CPUs=8 RealMemory=28000 State=CLOUD Weight=3".format(HN = ctrl_hostname)
 	C["NodeName1"] = "{HN}-worker[101-2000] CPUs=1 RealMemory=3000 State=CLOUD Weight=2".format(HN = ctrl_hostname)
 	C["NodeName4"] = "{HN}-worker[2001-3000] CPUs=4 RealMemory=23000 State=CLOUD Weight=4".format(HN = ctrl_hostname)
 	C["NodeName88"] = "{HN}-worker[3001-3500] CPUs=8 RealMemory=50000 State=CLOUD Weight=4".format(HN = ctrl_hostname)
-	C["NodeName99"] = "{HN}-nfs CPUs=4 RealMemory=14000 Weight=1".format(HN = ctrl_hostname)
+	C["NodeName89"] = "{HN}-worker[3501-3600] CPUs=8 RealMemory=50000 State=CLOUD Weight=4".format(HN = ctrl_hostname) # non-preemptible
```

Comment:
Do you think it would be valuable to create other node types in the non preemptible partition?

Reply:
I think it would be valuable. Getting some one-core machines might save some CPU time if certain jobs are finished early. Do you have a distribution of VMs in mind? I am thinking about getting some large VMs (32 or 64 core) as well - it would support jobs that requires more cores.